tfc-dump.pl

```diff
@@ -115,29 +115,39 @@
 my @vs_names;
 my @vs_ids;
 my $pg_size = 100;
+my $pg_num = 1;
 my $total_count;
 my $tmpfile = `mktemp`;
 chomp($tmpfile);
 
-$curl_query = "\"https://app.terraform.io/api/v2/organizations/${tfc_org_name}/varsets?page%5Bsize%5D=${pg_size}\"";
-$curl_cmd   = "curl $curl_headers --output $tmpfile $curl_query";
-system($curl_cmd);
-
-# Get the Variable Set names
-
-$jq_cmd   = "cat $tmpfile | jq '.data[].attributes.name'";
-@vs_names = `$jq_cmd`;
-# Remove the double quotes in each element of the array.
-grep($_ =~ s/"//g && 0, @vs_names);	# Programming Perl, p. 221, 1990
-chomp(@vs_names);
-
-# Get the Variable Set IDs
+do {
+	$curl_query = "\"https://app.terraform.io/api/v2/organizations/${tfc_org_name}/varsets?page%5Bsize%5D=${pg_size}&page%5number%5D=${pg_num}\"";
+	$curl_cmd   = "curl $curl_headers --output $tmpfile $curl_query";
+	system($curl_cmd);
```

Comment:
This is appropriate, because there's no need to capture output, but system returns 0 if successful. Error checking could be added, but this is not a requested nor required change.